learnr_bank/learnr_set1.Rmd

```diff
@@ -0,0 +1,386 @@
+100 == (-10)^2
+```
+
+``` {r, logical-q5-answer}
+#Solution to Question 5
+100 != -55
+```
+
+``` {r, logical-q6-answer}
+#Solution to Question 6
+!(100 < 88)
+```
+
+``` {r, logical-q7-answer}
+#Solution to Question 7
+100 == 100 & 66 == 66
+```
+
+``` {r, logical-q8-answer}
+#Solution to Question 8
+100 > 87  & 66 < 12
+```
+
+``` {r, logical-q9-answer}
+#Solution to Question 9
+100 > 87 | 66 < 12
+```
+### Set 3: Defining and understanding variables in R
+
+Please name these questions `variables_qX` where X is the number (1, 2, 3, ...). Note: for question 1 here, actually write a regular code chunk (just keep it as is!), and then only write the multiple choice. These should not be within in a `quiz()` - make an indiviual code chunk for each question (see the top example under "Questions" [here](https://blog.rstudio.com/2017/07/11/introducing-learnr/)).
+
+1. The code example below defines a variable. Answer the multiple choice questions about this code:
+```{r}
+y <- 8.5
+```
+
+```{r quiz1, echo=FALSE}
+quiz(caption = "Multiple Choice",
+  question("What is the *name* of this variable?",
+     answer("`<-`"),
+     answer("y", correct = TRUE),
+     answer("8.5"),
+     answer("We can't tell.")
+     ),
+  question("What is the *value* of this variable?",
+     answer("`<-`"),
+     answer("y"),
+     answer("8.5", correct = TRUE),
+     answer("We can't tell.")
+     ),
+  question("What is the *type* of this variable?",
+     answer("Character"),
+     answer("Numeric", correct = TRUE),
+     answer("Logical"),
+     answer("We can't tell."
+     )
+  )
+)
+```
+
+MC1: What is the *name* of this variable?
+a) `<-`
+b) y [CORRECT]
+c) 8.5
+d) We can't tell.
+
+MC2: What is the *value* of this variable?
+a) `<-`
+b) y
+c) 8.5 [CORRECT]
+d) We can't tell.
+
+MC3: What is the *type* of this variable?
+a) Character
+b) Numeric [CORRECT]
+c) Logical
+d) We can't tell.
+
+
+<!-- Shemaiah: for these questions, include the option:   exercise.lines=8     -->
+
+2. Using the code presented in question 1 to help guide you, define a variable named `banana_price` whose value is 0.55 (aka 55 cents). 
+  + **Hint:** Use the assignment operator `<-` to assign the value 0.55 to the variable name `banana_price`
+  + Solution: banana_price <- 0.55
+``` {r variables_q2, exercise = TRUE, exercise.lines=8}
+
+``` 
+
+<div id="logical_q2-hint">
+**Hint:**  Use the assignment operator `<-` to assign the value 0.55 to the variable name `banana_price`
+</div>
+
+3. Again, define the `banana_price` variable (since these questions don't communicate with each other!!). Use R as a calculator to determine the cost of 17 bananas.
+  + **Hint:** Perform this operation using the `banana_price` variable itself.
+  + Solution: 
+  ```
+  banana_price <- 0.55
+  banana_price * 17
+  ```
+``` {r variables_q3, exercise = TRUE, exercise.lines=8}
+
+``` 
+  
+<div id="logical_q3-hint">
+**Hint:**  Perform this operation using the `banana_price` variable itself.
```

Comment:
All of these hints are labeled with logical - therefore they are appearing with the logical questions in section 2. See the knitted output - just change the titles down here.